vnext/ReactWindowsCore/IWebSocket.h

```diff
@@ -73,41 +82,115 @@ struct IWebSocket
 
   #pragma endregion // Inner types
 
+  /// <summary>
+  /// Creates an <c>IWebSocket</c> instance.
+  /// </summary>
+  /// <param name="url">
+  /// WebSocket URL address the instance will connect to.
+  /// The address's scheme can be either ws:// or wss://.
+  /// </param>
   static std::unique_ptr<IWebSocket> Make(const std::string& url);
 
   // Only use if a legacy implementation is required as fallback.
   static std::unique_ptr<IWebSocket> MakeLegacy(const std::string& url);
 
   virtual ~IWebSocket() {}
 
-  ///
-  // Parameters:
-  //
-  // protocols - vector of protocols
-  // options   - map of headers
-  ///
+  /// <summary>
+  /// Establishes a continuous connection with the remote endpoint.
+  /// </summary>
+  /// <param name="protocols">
+  /// Currently unused
+  /// </param>
+  /// <param name="options">
+  /// HTTP header fields passed by the remote endpoint, to be used in the handshake process.
+  /// </param>
   virtual void Connect(const Protocols& protocols = {}, const Options& options = {}) = 0;
+
+  /// <summary>
+  /// Sends a ping frame to the remote endpoint.
+  /// </summary>
   virtual void Ping() = 0;
+
+  /// <summary>
+  /// Sends a text message to the remote endpoint.
+  /// </summary>
+  /// <param name="message">
+  /// UTF8-encoded string of arbitrary length.
+  /// </param>
   virtual void Send(const std::string& message) = 0;
+
+  /// <summary>
+  /// Sends a non-plain-text message to the remote endpoint.
+  /// </summary>
+  /// <param name="base64String">
+  /// Binary message encoded in Base64 format.
+  /// </param>
   virtual void SendBinary(const std::string& base64String) = 0;
+
+  /// <summary>
+  /// Terminates this resource's connection to the remote endpoint.
+  /// This instance can't be restarted or re-connected afterwards.
+  /// </summary>
+  /// <param name="close">
+  /// </param>
+  /// <param name="reason">
+  /// </param>
   virtual void Close(CloseCode code, const std::string& reason) = 0;
 
+  /// <returns>
+  /// Current public state as defined in the <c>ReadyState</c> enum.
+  /// </returns>
   virtual ReadyState GetReadyState() const = 0;
 
+  /// <summary>
+  /// Sets the optional custom behavior on a successful connection.
+  /// </summary>
+  /// <param name="handler">
```

Comment:
><param name="handler"> [](start = 6, length = 22)

[nit] Description of 'handler' parm [here and below]?

Reply:
Will add in a subsequent change.